contrib/IECoreAlembic/src/IECoreAlembic/AlembicScene.cpp

```diff
@@ -834,9 +835,13 @@ class AlembicScene::AlembicWriter : public AlembicIO
 			{
 				sample.setMatrix( matrixData->readable() );
 			}
+			else if( const TransformationMatrixdData *transformationMatrixData = runTimeCast<const TransformationMatrixdData>( transform ) )
+			{
+				sample.setMatrix( transformationMatrixData->readable().transform() );
```

Comment:
I'm happy to do it this way, but I thought Alembic does have some notion of a more Maya-like Xform. Should we try preserving all that jazz?